websites/Y/YomuMangás/presence.ts

```diff
@@ -0,0 +1,134 @@
+			}${chapterTitle ? ` - ${chapterTitle}` : ""}`;
+		}
+		if (!chapter && isReader) presenceData.state = "Lendo Capítulo";
+		if (manga && ((isAdult && adult) || !isAdult) && !(chapter && isReader)) {
+			presenceData.buttons = [
+				{
+					label: "Ver Mangá",
+					url: document.querySelector<HTMLAnchorElement>("#premid-manga-url")
+						?.href,
+				},
+			];
+		}
+		if (manga && ((isAdult && adult) || !isAdult) && chapter && isReader) {
+			presenceData.buttons = [
+				{
+					label: "Ver Mangá",
+					url: document.querySelector<HTMLAnchorElement>("#premid-manga-url")
+						?.href,
+				},
+				{
+					label: "Ver Capítulo",
+					url: document.querySelector<HTMLAnchorElement>("#premid-chapter-url")
+						?.href,
+				},
+			];
+		}
+	} else if (pathname.includes("/user")) {
+		presenceData.details = user
+			? `Perfil de ${document.querySelector("#premid-user-name")?.textContent}`
+			: "Stalkeando Alguém";
+		presenceData.buttons = [
+			{
+				label: "Ver Perfil",
+				url: document.querySelector<HTMLAnchorElement>("#premid-user-url")
+					?.href,
+			},
+		];
+		if (images && user) {
+			presenceData.largeImageKey =
+				document.querySelector<HTMLAnchorElement>("#premid-user-image")?.href;
+		}
+	} else if (pathname.includes("/signup"))
+		presenceData.details = "Registrando-se";
+	else if (pathname.includes("/recovery"))
+		presenceData.details = "Recuperando a Conta";
+	else if (pathname.includes("/login")) presenceData.details = "Fazendo Login";
+	else if (pathname.includes("/404")) {
+		presenceData.details = "404";
+		presenceData.state = "Perdido no Mato";
+	} else if (pathname.includes("/tutorial"))
+		presenceData.details = "Lendo o Tutorial";
+	else if (pathname.includes("/faq")) presenceData.details = "Lendo o FAQ";
+	else if (pathname.includes("/terms-of-use"))
+		presenceData.details = "Lendo os Termos de Uso";
+	else if (pathname.includes("/privacy"))
+		presenceData.details = "Lendo as Políticas de Privacidade";
+	else if (pathname.includes("/dmca")) presenceData.details = "Lendo o DMCA";
+	else if (pathname.includes("/search"))
+		presenceData.details = "Busca Avançada";
+	else if (pathname.includes("/about")) presenceData.details = "Sobra a Yomu";
+	else if (pathname.includes("/donate"))
+		presenceData.details = "Página de Doação";
+	else if (pathname.includes("/contact"))
+		presenceData.details = "Página de Contato";
+	else if (pathname.includes("/ranks"))
+		presenceData.details = "Página de Ranks";
+	else if (pathname.includes("/minigames"))
+		presenceData.details = "Página de Minigames";
+	else if (pathname.includes("/donators"))
+		presenceData.details = "Página de Doadores";
+	else if (pathname.includes("/staffs"))
+		presenceData.details = "Página da Equipe";
+	else if (pathname.includes("/partners"))
+		presenceData.details = "Página de Parceiros";
+	else if (pathname.includes("/settings"))
+		presenceData.details = "Página de Configurações";
```

Comment:
> Add the static pages to a Record, shown in the example here
> 
> https://github.com/PreMiD/Presences/blob/cf3631226067d77ada628adbf77d7d5e5e2ca445/websites/T/TruckersMP/presence.ts#L15-L41

Reply:
Done, now the static presence pages are a Record